### PR TITLE
Implement OperateData

### PR DESCRIPTION
**Pull request by Jazz86**

### CODE REVIEW

File: src/Implement/UnitTestOperateData.cpp [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  EXPECT_EQ("136", test2.header["Content-Length"]);
+  EXPECT_EQ("test/myFile/filename.txt", test2["path"]);
+  EXPECT_EQ("0", test2["pos"]);
+  EXPECT_EQ("123456789", test2["data"]);
+}
+
+int main(int argc, char **argv) {
+  testing::InitGoogleTest(&argc, argv);
+  return RUN_ALL_TESTS();
+}
```
Comment (younger15) [inline src/Implement/UnitTestOperateData.cpp]: If you're going to do unit test with one main then the main function here should remove

Comment (Jazz86) [reply src/Implement/UnitTestOperateData.cpp]: done. The main function has been removed.

Comment (john0312) [reply src/Implement/UnitTestOperateData.cpp]: This is a reasonable review.

File: src/Implement/UnitTestOperateData.cpp [hunk 2]
```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2021, 鍾淯丞, 周杰仕, 林仁鴻. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gtest/gtest.h>
+
+#include "OperateData.h"
+
+// test basic usage
+TEST(OPERATEDATA_TEST, Basic) {
```
Comment (john0312) [inline src/Implement/UnitTestOperateData.cpp:10]: [Optional] Sometimes we call this the "smoke test".

**Review by john0312 — APPROVED** [review]

**Review by younger15 — APPROVED** [review]

**Pull request merged**